iina/Extensions.swift

```diff
@@ -173,6 +183,12 @@ extension NSMenu {
   }
 }
 
+extension String {
```

Comment:
No, it just splits the string (should be the same as `components(separatedBy:)`?)

Or I can just use `components(separatedBy:)`.